Sources/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -518,6 +571,7 @@ - (CGRect)getViewBounds {
 }
 
 - (CGSize)getSourceDimensions {
+    // TODO: What throws here? Is this try-catch even necessary
```

Comment:
Probably since it manually iterates over `_player.currentItem.tracks` which means any of `_player`, `currentItem`, or `tracks` could all change mid-iteration. Using "fast iteration" takes care of this - behind the scenes it grabs an `objectEnumerator` property off the collection one time and uses that to implement the `for in` loop. `enumerateObjectsUsingBlock:` has the same benefit if you still need the index for some reason.
In any case I've flagged this for replacement in another ticket.